src/sentry/issues/auto_source_code_config/code_mapping.py

```diff
@@ -12,15 +12,13 @@
     RepoTreesIntegration,
     get_extension,
 )
-from sentry.issues.auto_source_code_config.constants import (
-    EXTRACT_FILENAME_FROM_MODULE_AND_ABS_PATH,
```

Comment:
This information can now be determined via `platform_config.extracts_filename_from_module()`